src/main/java/io/gravitee/policy/xslt/XSLTTransformationPolicy.java

```diff
@@ -114,6 +114,11 @@ public ReadWriteStream onRequestContent(Request request, ExecutionContext execut
                     xsltTransformationPolicyConfiguration.getParameters().forEach(
                             parameter -> {
                                 if (parameter.getName() != null && ! parameter.getName().trim().isEmpty()) {
+                                    // Apply SpEL conversion
+                                    Object extValue = (parameter.getValue() != null) ? executionContext.getTemplateEngine().getValue(parameter.getValue(), String.class) : null;
```

Comment:
Sorry, probably my fault, but it must be:

```suggestion
                                    String extValue = (parameter.getValue() != null) ? executionContext.getTemplateEngine().getValue(parameter.getValue(), String.class) : null;
```